README.md

```diff
@@ -612,6 +613,7 @@ Services to securely store your Docker images.
 
 * [Contributing to Docker by Andrew "Tianon" Page (InfoSiftr)](https://www.youtube.com/watch?v=1jwo8-1HYYg) (34:31)
 * [Deploying and scaling applications with Docker, Swarm, and a tiny bit of Python magic](https://www.youtube.com/watch?v=GpHMTR7P2Ms) (3:11:06) by [@jpetazzo][jpetazzo]
+* [Dive Into Docker: The Complete Docker Course For Developers](https://www.youtube.com/watch?v=XeSD17YRijk&list=PL-v3vdeWVEsXT-u0JDQZnM90feU3NE3v8) (60:25) by [@nickjanetakis](https://twitter.com/nickjanetakis)
```

Comment:
This is a total beginner course. It seems complete though. I would rather put it into the "Where to start?" section

Reply:
Good idea. I've moved it to that section.